cmd/restic/integration_test.go

```diff
@@ -1,3 +1,72 @@
-// build +integration
+// +build integration
 
-package main_test
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	. "github.com/restic/restic/test"
+)
+
+var TestDataFile = flag.String("test.datafile", "", `specify tar.gz file with test data to backup and restore (required for integration test)`)
+
+func cmd_init(t testing.TB) {
+	cmd := &CmdInit{}
+	OK(t, cmd.Execute(nil))
+
+	t.Logf("repository initialized at %v", opts.Repo)
+}
+
+func cmd_backup(t testing.TB, target []string) {
+	cmd := &CmdBackup{}
+	t.Logf("backing up %v", target)
+
+	OK(t, cmd.Execute(target))
+}
+
+func system(command string, args ...string) error {
+	cmd := exec.Command(command, args...)
+	cmd.Stdout = os.Stdout
```

Comment:
Right, sorry, my bad. Maybe we should stop using `fmt` and instead add something that implements the logger interface, so that in test, we can inject a null logger. I can work on that if you want.